tools/ci.sh

```diff
@@ -249,6 +249,7 @@ function ci_qemu_arm_build {
 
 function ci_renesas_ra_setup {
     ci_gcc_arm_setup
+    sudo apt-get install protobuf-compiler
```

Comment:
I don't think this change is needed, because the CI does not build any boards that use the `esp_hosted` driver.

Reply:
Where do I add this dependency to make sure the firmware is built and hosted on micropython.org ? 
BTW it might be a good idea to also add this board to the CI, since it uses the most features (wifi, bt, USB).

Reply:
Add in a new commit.

Reply:
It seems the protobuf compiler needs to be added first in a different commit, but I think that last commit groups the changes to `ci.sh` better and should be last, should I try to fix this or leave it ?

Reply:
As you have it now is fine, so you can leave it.

Reply:
The CI is failing.

Reply:
Should be fixed now, it was the wrong protobuf command name, but unix port is failing probably an unrelated error.